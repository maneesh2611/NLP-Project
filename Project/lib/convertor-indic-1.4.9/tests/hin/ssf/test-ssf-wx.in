1	rAma
2	ਰਹ੍ਯਿਣਾ
3	రాశిద్
4	jAwA
5	hE.
6	60vAM
7	राशिद
