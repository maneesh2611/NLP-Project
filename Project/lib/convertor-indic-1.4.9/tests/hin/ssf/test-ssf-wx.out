1	राम
2	ਰਹ੍ਯਿਣਾ
3	రాశిద్
4	जाता
5	है.
6	60वां
7	राशिद
